ui/raidboss/data/07-dt/trial/queen-eternal-ex.txt

```diff
@@ -0,0 +1,211 @@
+# -p A04B:1000.0 -ii A008 A009 A00D A00E A00F A010 A018 A026 A027 A02F A030 A053 A054 A056
+
+# Phase one pulled from -r XwHyk2NAKnbzZLf7 -rf 3
+# Phase two pulled from -r GBaArv7yzHnXck8D -rf 17
+# A lot of manual cleanup applied
+
+# Phase one
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+12.9 "--sync--" StartsUsing { id: "A00C", source: "Queen Eternal" }
+15.6 "Aethertithe" Ability { id: "A00C", source: "Queen Eternal" } duration 27.8
+27.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+28.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+36.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+37.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+45.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+46.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+58.7 "Virtual Shift" Ability { id: "A019", source: "Queen Eternal" }
+
+# Wind miniphase
+67.8 "Laws of Wind" Ability { id: "A023", source: "Queen Eternal" }
+72.8 "Aeroquell" Ability { id: "A025", source: "Queen Eternal" }
+85.8 "Legitimate Force" Ability { id: ["A01E", "A020"], source: "Queen Eternal" }
+88.8 "Legitimate Force" Ability { id: ["A021", "A022"], source: "Queen Eternal" }
```

Comment:
```suggestion
85.8 "Legitimate Force 1" Ability { id: ["A01E", "A020"], source: "Queen Eternal" }
88.8 "Legitimate Force 2" Ability { id: ["A021", "A022"], source: "Queen Eternal" }
```

Feel free to ignore here and elsewhere, but I usually label sequences where it's not just a repeated action like raidwides.

Reply:
This is a left/right room cleave. So maybe this instead if we want to differentiate? But there's also a trigger for left/right as well...

```
85.8 "Legitimate Force Left/Right" Ability { id: ["A01E", "A020"], source: "Queen Eternal" }
88.8 "Legitimate Force Right/Left" Ability { id: ["A021", "A022"], source: "Queen Eternal" }
```

Reply:
Mm, you're right, that's probably better. If you're comfortable with it, let's do that.